.github/workflows/ubuntu-latest.yml

```diff
@@ -66,3 +66,4 @@ jobs:
     # Update the website to show new version.
     if: ${{ startsWith(github.ref, 'refs/tags/v') }}
     uses: ./.github/workflows/update-website.yml
+    secrets: inherit
```

Comment:
@CAMOBAP by the looks of it, your editor is not configured to not leave trailing newline? I think we generally use trailing newlines, it’s a good idea to stick to one approach to avoid unnecessary diffs I guess